styles/product.css

```diff
@@ -0,0 +1,38 @@
+.product-img {
+  width: 35rem;
+  height: 40rem;
```

Comment:
instead of this give only one thing wether width or height and then use object-fit : cover, it will take care of responsiveness

Reply:
Sure will add that